connector/connector.py

```diff
@@ -166,6 +170,22 @@ def match(cls, session, model):
             model_name = model  # str
         return model_name in cls.model_name
 
+    @property
+    def env(self):
+        """ Returns the openerp.api.environment """
+        return self.session.env
+
+    @property
+    def model(self):
+        _logger.warning('Deprecated: ConnectorUnit.model is no longer a '
+                        'property. Now, you should use '
+                        'ConnectorUnit.recordset() that returns a '
+                        'new-api model/recordset.')
+        return self.session.pool.get(self.environment.model_name)
+
+    def recordset(self, model=None):
```

Comment:
The perfect name for this method would have been `model`... but it is already used.
`recordset` is a bit odd as it returns a model (but a model is also a new recordset in the new api), but I can't find a better name.